src/provider.tsx

```diff
@@ -92,14 +94,14 @@ export const IntercomProvider = ({
   }, []);
 
   const shutdown = React.useCallback(() => {
-    if (!isBooted.current) return;
+    if (isSSR || !isBooted.current) return;
 
     IntercomAPI('shutdown');
     isBooted.current = false;
   }, []);
 
   const hardShutdown = React.useCallback(() => {
-    if (!isBooted.current) return;
+    if (isSSR || !isBooted.current) return;
```

Comment:
What is the specific reason for this check? It seems there is no use case such as: 
"Intercom would be booted in a web environment and then suddenly wants to hard shutdown in a "not" web environment"?

Reply:
See comment above